client/js/components/BeerCarousel/BeerCarousel.js

```diff
@@ -0,0 +1,202 @@
+import React, { Component } from 'react';
+import { Dimensions, Image, Text, View, TouchableOpacity } from 'react-native';
+import Carousel from 'react-native-snap-carousel';
+
+import styles from './styles';
+import { hr, center, h2, h3, Subtitle1 } from '../../config/styles';
+
+// onPress={() => {
+//   this._carousel.snapToNext();
+// }}
+
+export class BeerCarousel extends Component {
+  _renderItem({ item, index }) {
+    return (
+      <View style={styles.slide}>
+        <Text style={styles.title}>{item.title}</Text>
+      </View>
+    );
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      slider1ActiveSlide: SLIDER_1_FIRST_ITEM
+    };
+  }
+
+  _renderItem({ item, index }) {
+    return <SliderEntry data={item} even={(index + 1) % 2 === 0} />;
+  }
+
+  _renderItemWithParallax({ item, index }, parallaxProps) {
+    return (
+      <SliderEntry
+        data={item}
+        even={(index + 1) % 2 === 0}
+        parallax={true}
+        parallaxProps={parallaxProps}
+      />
+    );
+  }
+
+  _renderLightItem({ item, index }) {
+    return <SliderEntry data={item} even={false} />;
+  }
+
+  _renderDarkItem({ item, index }) {
+    return <SliderEntry data={item} even={true} />;
+  }
+
+  mainExample(number, title) {
```

Comment:
looks like you copied this file from the docs and didn't remove unnecessary code. Please make the changes.